distribution/Makefile

```diff
@@ -46,14 +91,14 @@ aws-cloud-container-check: containers container-check-prereqs
 
 container: container-prereqs Dockerfile config/supervisord.conf
 	$(MAKE) -C client build
-	docker build -t ${IMAGE_NAME}:latest .
+	docker build -t ${IMAGE_NAME}:base .
 	$(MAKE) -C environments $@
 
 containers: container
 	$(MAKE) -C ../services container
 
 publish: container
-	docker push ${IMAGE_NAME}:latest
+	docker push ${IMAGE_NAME}:base
```

Comment:
I do not even think we should push this actually. I _think_ Docker does not need an inherited image to be pushed too, it will just push the layers for inheriting images.